symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/stream/util/StreamUtil.java

```diff
@@ -0,0 +1,45 @@
+package com.symphony.bdk.core.service.stream.util;
+
+import org.apiguardian.api.API;
+
+import java.util.Base64;
+
+/**
+ * Helper class providing stream id conversion
+ */
+@API(status = API.Status.EXPERIMENTAL)
+public class StreamUtil {
+
+  private StreamUtil() {
+  }
+
+  /**
+   * Convert the stream id to the corresponding URLSafe encoded stream id
+   *
+   * @param streamId of the stream to be parsed
+   * @return stream id after conversion
+   */
+  public static String toUrlSafeStreamId(String streamId) {
+    if (streamId == null || streamId.isEmpty()) {
```

Comment:
i would be ok to throw an NPE is null is passed, then we can remove the isEmpty check too